port/unix/omrfile.c

```diff
@@ -353,7 +356,7 @@ omrfile_open(struct OMRPortLibrary *portLibrary, const char *path, int32_t flags
 #endif
 
 	if (-1 == fd) {
-		Trc_PRT_file_open_Exception2(path, errno, findError(errno));
```

Comment:
No, didn't mean to remove it, nice catch.  Yes common code updates in this pull would affect other platforms, which is why I did a personal VM build.  The segments under z/TPF directory shouldn't affect other platforms.